code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -804,6 +803,14 @@
 	safe_plasma_min = 4 //We breathe THIS!
 	safe_plasma_max = 0
 
+/obj/item/organ/internal/lungs/plasmaman/plasmaman_smoker
+	name = "smoker plasma filter"
+	desc = "A plasma filter that look discolored, a result from smoking a lot."
+	icon_state = "lungs-plasma-s"
```

Comment:
There's no strong reason you couldn't call the icon state `lungs-plasma-smoker`, it's clearer and with no downsides.